src/components/AppToolbar/AppToolbar.tsx

```diff
@@ -69,6 +77,17 @@ const AppToolbar: React.FC<AppToolbarProps> = ({ open, handleDrawerOpen }) => {
       marginLeft: 'auto',
       margin: theme.spacing(0, 0.5),
     },
+    logoContainer: {
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      marginRight: 24,
+      ...theme.mixins.toolbar,
+    },
+    logo: {
+      // Ensures min 1/4 of the height is padding
+      maxHeight: '71.4%',
```

Comment:
This is a bit ugly, but I struggled to find another way to do this. The space around the image needs to be 20% of the image's height per UKRI rules on logo use. When using padding with percentages, it's calculated from the width of the available space rather than height. I wanted to get the height of the toolbar and calculate it based on that, e.g.

``padding: `calc(${theme.mixins.toolbar.height} * 0.2)` ``

but the toolbar height just kept coming back as undefined.